implementation/choreography-to-orchestration/lambda/orchestration/ProcessPayment.py

```diff
@@ -0,0 +1,8 @@
+import json
+
+def lambda_handler(event, context):
```

Comment:
Per the diagram, patterns/images/choreography-to-orchestration/orchestration.png, the Lambda stores data. Looks like the implementation is missing that

Reply:
Added Storage(DynamoDB table)

Reply:
It is still missing from the lambda